backend/tools/scrape.py

```
@@ -0,0 +1,358 @@
+        if minPrice > 0:
+            params['minPrice'] = minPrice
+        if maxPrice > 0:
+            params['maxPrice'] = maxPrice
+        
+        # 색상 조건이 있을 경우에만 파라미터에 추가
+        if color.strip():
+            params['color'] = color
+        if shoeSize:
+            params['shoeSizeOption'] = shoeSize
+
+        url = f"https://api.musinsa.com/api2/dp/v1/plp/goods"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)